src/random.cr

```diff
@@ -1,66 +1,125 @@
 require "random/mt19937"
 
+# `Random` provides an interface for random values generation, using a pseudo random number generator (PRNG).
+# 
+# ```
+# Random.new_seed # => 112705036
+# Random.rand     # => 0.167595
+# Random.rand(5)  # => 2
+# ```
+#
+# The above methods delegate to a shared instance. You can create one like:
```

Comment:
That sounds like it would create a new *shared* instance, which is not quite right.